omod/src/main/webapp/resources/htmlforms/sierra_leone/vitals.xml

```diff
@@ -549,9 +549,47 @@
             </fieldset>
 
         </includeIf>
+<fieldset>
+	<p>
+            <legend><uimessage code="mirebalais.vitals.response.title"/></legend>
+<!--taken from haiti/hiv/isanteplus VisiteDeSuivi.xml -->
+            <h3><uimessage code="mirebalais.vitals.response.title"/></h3>
+<table width="100%">
+		<tr>
+			<td width="50%" valign="TOP">
+				<section> 
+<!--					 <table>
+						
+    						   <tr>
+     						       <obsgroup groupingConceptId="PIH:10724">
+     						        <td><obs conceptId="CIEL:160282" labelText="Awake (alert)" style="radio"/></td>
+      						        <td><obs conceptId="CIEL:162645" labelText="Responds to voice" style="radio"/></td>
+							<td><obs conceptId="CIEL:162644" labelText="Responds to voice" style="radio"/></td>
+							<td><obs conceptId="PIH:6784" labelText="Staring and unresponsive" style="radio"/></td>
+    						        </obsgroup>
+    						    </tr>
+    					</table> -->
+					<table>
+						
+						
+						<tr>
+							<td class="checkboxGroup"><!-- formatting changed from original concepts changed and translated -->
+								&#160;<obs conceptId="PIH:10674" id="response_level" answerConceptIds="CIEL:160282,CIEL:162645,CIEL:162644,PIH:6784" answerLabels="Awake (alert),Responds to voice,Responds to pain,Staring and unresponsive" style="radio"/>
```

Comment:
Please break up long lines into multiple smaller ones. This could be
```
&#160;
<obs conceptId="PIH:10674" id="response_level"
    answerConceptIds="CIEL:160282,CIEL:162645,CIEL:162644,PIH:6784"
    answerLabels="Awake (alert),Responds to voice,Responds to pain,Staring and unresponsive"
    style="radio" />
```